src/audio/AudioStream.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @author Mugen87 / https://github.com/Mugen87
+ */
+
+function AudioStream( path, listener ) {
```

Comment:
I've tried to address your feedback in the latest commit. I removed `path` from the constructor and added a new method `setAudioElement()`.